base_synchro/wizard/base_synchro.py

```diff
@@ -297,19 +303,24 @@ def upload_download(self):
 
     @api.multi
     def upload_download_multi_thread(self):
-        threaded_synchronization = \
-            threading.Thread(target=self.upload_download())
-        threaded_synchronization.run()
-        data_obj = self.env['ir.model.data']
-        id2 = data_obj._get_id('base_synchro', 'view_base_synchro_finish')
-        if id2:
-            id2 = data_obj.browse(id2).res_id
-        return {
-            'view_type': 'form',
-            'view_mode': 'form',
-            'res_model': 'base.synchro',
-            'views': [(id2, 'form')],
-            'view_id': False,
-            'type': 'ir.actions.act_window',
-            'target': 'new',
-        }
+        try:
+            threaded_synchronization = \
+                threading.Thread(target=self.upload_download())
+            threaded_synchronization.run()
+            data_obj = self.env['ir.model.data']
+            id2 = data_obj._get_id('base_synchro', 'view_base_synchro_finish')
+            if id2:
+                id2 = data_obj.browse(id2).res_id
+            return {
+                'view_type': 'form',
+                'view_mode': 'form',
+                'res_model': 'base.synchro',
+                'views': [(id2, 'form')],
+                'view_id': False,
+                'type': 'ir.actions.act_window',
+                'target': 'new',
+            }
+        except:
+            raise Warning(_('''Synchronization cannot be done because
+            in destination database model is not found or reference is not
+            correctly set!'''))
```

Comment:
here, logger is good.

Reply:
Sir, here I have not written warning in logger because if a model is found in one database and that model is not found in other db, so Warning is raised and sync will not be done because that model is not found in other db.